solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1430,6 +1433,48 @@ impl pallet_data_preservers::Config for Runtime {
     type MaxParaIdsVecLen = MaxLengthParaIds;
 }
 
+parameter_types! {
+    pub ParachainBondAccount: AccountId32 = PalletId(*b"ParaBond").into_account_truncating();
```

Comment:
Let's change this naming. StarlightBondAccount or something similar should do it. Or we wire it to treasury.

Reply:
I think we should follow the same approach as we have in Dancebox, and not wire the Treasury account here for now, just in case we need a clearer (and separated) vision of the 30% that goes to this account.

I renamed the account here becc74009348a33015b3290fbfe6e5ffb5b5a816.

Let me know if you don't agree and in that case we can discuss a bit more about it :)

Reply:
_roadmap_